Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/AddOrEditResourcesOnClaimSetCommand.cs

```diff
@@ -69,16 +71,22 @@ public void Execute(int claimSetId, List<ResourceClaim> resources)
                     AuthorizationStrategyForCreate = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.Create()),
                     AuthorizationStrategyForRead = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.Read()),
                     AuthorizationStrategyForUpdate = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.Update()),
-                    AuthorizationStrategyForDelete = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.Delete()),
+                    AuthorizationStrategyForDelete = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.Delete()),
                     AuthorizationStrategyForReadChanges = AuthStrategyOverrideForAction(resource.AuthStrategyOverridesForCRUD.ReadChanges())
                 };
                 _overrideDefaultAuthorizationStrategyCommand.Execute(overrideAuthStrategyModel);
             }
         }
 
-        static int AuthStrategyOverrideForAction(AuthorizationStrategy? authorizationStrategy)
-        {
-            return authorizationStrategy != null ? authorizationStrategy.AuthStrategyId : 0;
+        static int[] AuthStrategyOverrideForAction(ClaimSetResourceClaimActionAuthStrategies? claimSetResourceClaimActionAuthStrategies)
+        {
+            if (claimSetResourceClaimActionAuthStrategies != null && claimSetResourceClaimActionAuthStrategies.AuthorizationStrategies != null)
+            {
+#pragma warning disable CS8602 // Dereference of a possibly null reference.
+                return claimSetResourceClaimActionAuthStrategies.AuthorizationStrategies.Where(p => p is not null).Select(p => p.AuthStrategyId).ToArray();
+#pragma warning restore CS8602 // Dereference of a possibly null reference.

```

Comment:
@jagudelo-gap 
 Select(p => p!.AuthStrategyId).ToArray(); adding null-forgiving operator after "p" will fix the warning. Please remove warning disable.